packages/web/src/pages/invalidEmail.tsx

```diff
@@ -0,0 +1,35 @@
+import { NextPage } from 'next';
+import { Button, Heading, HStack, Text, VStack } from '@chakra-ui/react';
+import { MarketingLayout } from '../components/Layout';
+
+const InvalidEmail: NextPage = () => {
+  return (
+    <MarketingLayout>
+      <VStack alignItems="center" paddingTop={8} w="100%" maxW="1200px" marginX="auto">
+        <Heading size="2xl" color="red" textAlign="center" lineHeight="1.4">
```

Comment:
This is a little aggressive; let's tone it down a bit 😉 
```suggestion
        <Heading size="lg" textAlign="center">
```